modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -28,6 +28,14 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+// todo: instead of creating a texture and discarding it / instead create an image and use it directly instead of pointlessly instancing
+// todo: memory leak on spatial node / not being cleaned up properly (below)
+/* todo: fix WARNING: cleanup: Leaked instance: Spatial:31345 - Node name:
+* ObjectDB Instances still exist!
+*   At: core/object.cpp:2098.
+*/
+// why? performance benefit 2x - directly fetching from visual server, instead of keeping the one we already have available
+
 #include "assimp/DefaultLogger.hpp"
```

Comment:
These assimp includes should also use `#include <assimp/Importer.hpp>`.

Reply:
Fixed